etc/opensc.conf

```diff
@@ -1,4 +1,24 @@
 app default {
 	# debug = 3;
 	# debug_file = opensc-debug.txt;
+	framework pkcs15 {
+        pkcs15init PIV-II {
+            module = "C:\Program Files\OpenSC Project\OpenSC\ishield_piv_module.dll";
+        }
+    };
+    card_atr 3b:97:11:81:21:75:69:53:68:69:65:6c:64:05 {
+        driver = "PIV-II";
+    };
+	card_atr 3b:87:80:01:69:53:68:69:65:6c:64:50 {
+        driver = "PIV-II";
+    };
+	card_atr 3b:89:80:01:53:43:45:2e:37:20:20:20:20:44 {
+        driver = "PIV-II";
+    };
+	card_atr 3b:d5:18:ff:81:b1:fe:45:1f:c3:80:73:c8:21:10:6f {
+        driver = "PIV-II";
+    };
+	card_atr 3b:85:80:01:80:73:c8:21:10:0e {
+        driver = "PIV-II";
+    };
```

Comment:
Let's only include ATRs that are also added to `win32/customactions.cpp`. Could you please take a look at the formatting of the conf?